responders/src/UI/embc-responder/src/app/shared/components/dialog-components/information-dialog/information-dialog.component.ts

```diff
@@ -7,7 +7,7 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 })
 export class InformationDialogComponent implements OnInit {
   @Output() outputEvent = new EventEmitter<string>();
-  @Input() inputEvent: string;
+  @Input() inputEvent: any;
```

Comment:
'Any' should not be used. Please use appropriate type. Code cleanup is already in progress to remove 'any' from codebase